iis-workflows/pom.xml

```diff
@@ -113,6 +115,34 @@
 				<plugins>
 				
 					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-enforcer-plugin</artifactId>
+						<version>1.4.1</version>
+						<executions>
+							<execution>
+								<id>enforce-connection-properties-file-existence</id>
+								<phase>initialize</phase>
+								<goals>
+									<goal>enforce</goal>
+								</goals>
+								<configuration>
+									<rules>
+										<requireFilesExist>
+											<files>
+												<file>${iisConnectionProperties}</file>
+											</files>
+											<message>
+												Connection properties file doesn't exist in ${iisConnectionProperties}. Please create this file or override it's location with -DiisConnectionProperties property.
```

Comment:
I find this description not precise enough. Maybe:
The file with connection properties could not be found. Please, create the ${iisConnectionProperties} file or set the location to already created file by using -DiisConnectionProperties property.